Readme.md

```diff
@@ -3,22 +3,20 @@
 [![](https://img.shields.io/badge/Open_in_DevExpress_Support_Center-FF7200?style=flat-square&logo=DevExpress&logoColor=white)](https://supportcenter.devexpress.com/ticket/details/E813)
 [![](https://img.shields.io/badge/📖_How_to_use_DevExpress_Examples-e9f6fc?style=flat-square)](https://docs.devexpress.com/GeneralInformation/403183)
 <!-- default badges end -->
-<!-- default file list -->
-*Files to look at*:
-
-* [Form1.cs](./CS/Thread/Form1.cs) (VB: [Form1.vb](./VB/Thread/Form1.vb))
-* [Program.cs](./CS/Thread/Program.cs) (VB: [Program.vb](./VB/Thread/Program.vb))
-<!-- default file list end -->
-# Updating a Grid datasource from a separate thread
+# Data Grid for Windows Forms - Update a Grid Control's datasource from a separate thread
 
 
-<p>This example shows how to properly update the Grid's underlying data source from a background thread, and avoid possible synchronization problems.</p><p><strong>See Also:</strong><br />
-<a href="https://www.devexpress.com/Support/Center/p/AK2981">Can I avoid the ArgumentOutOfRangeException when updating the Grid's data</a></p>
+This example shows how to safely update a [Grid Control](https://docs.devexpress.com/WindowsForms/DevExpress.XtraGrid.GridControl)'s data source from a background thread. The approach shown allows you to avoid possible synchronization issues between the main and background threads.
```

Comment:
```suggestion
This example shows how to safely update a [Grid Control](https://docs.devexpress.com/WindowsForms/DevExpress.XtraGrid.GridControl)'s data source from a background thread. The technique shown allows you to avoid possible synchronization issues between the main and background threads.
```